eu.nokun.MirrorHall.json

```diff
@@ -51,10 +51,42 @@
         "python3-requirements.json",
         {
             "name": "inetutils",
-            "buildsystem": "simple",
-            "build-commands": [
-                "./configure --prefix=/app --libexec=/app/bin --localstatedir=/var --sysconfdir=/etc --without-{wrap,pam} --disable-{dnsdomainname,ftp,ftpd,ifconfig,inetd,logger,ping,ping6} --disable-{rcp,rexec,rexecd,rlogin,rlogind,rsh,rshd,syslogd} --disable-{talk,talkd,telnet,telnetd,tftp,tftpd,traceroute,uucpd,whois} --enable-hostname",
-                "make -j ${FLATPAK_BUILDER_N_JOBS}",
+            "buildsystem": "autotools",
+            "config-opts": [
+                "--prefix=/app",
+                "--libexec=/app/bin",
+                "--localstatedir=/var",
+                "--sysconfdir=/etc",
```

Comment:
there is no writable `/etc` so this may fail

Reply:
Do you know if I can safely remove it in this case?